### DISCUSSION TITLE
fast charging

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2646] [General]
Since I don't know much about the type of battery integrated on pybamm, is there a high-power battery suitable for fast charging experiments?

**Comment by brosaplanella** [comment]
I am not sure we have any parameter sets for high-power battery. I would suggest going through the various [parameter sets](https://pybamm.readthedocs.io/en/latest/source/api/parameters/parameter_sets.html#bundled-parameter-sets) and checking the references to see if any are for a high-power battery.

**Reply by RobinBluehub** [reply]
I tried all the battery types integrated on pybamm, and found that only the Prada2013 battery parameter set can meet the fast charging conditions. But unfortunately there may be some problems with this parameter set.
